package.json

```diff
@@ -34,32 +34,32 @@
     "tslint": "tslint --fix --project .",
     "prettier": "prettier --write lib/*.ts test/*.js",
     "prepare": "tsc",
-    "test": "tap --no-ts --node-arg=-r --node-arg=ts-node/register --no-coverage test/*_test.js"
+    "test": "tap --node-arg=-r --node-arg=ts-node/register --disable-coverage test/*_test.js"
   },
   "devDependencies": {
-    "@types/node": "^13.11.1",
-    "@types/stack-trace": "0.0.29",
-    "@types/uuid": "^7.0.2",
-    "express": "^4.17.1",
-    "http-terminator": "^2.0.3",
-    "jshint": "^2.5.6",
-    "nock": "~9",
-    "prettier": "^2.0.4",
-    "semver": "^5.0.3",
-    "tap": "^15.0.9",
-    "ts-node": "^8.8.2",
+    "@types/node": "^20.12.7",
+    "@types/stack-trace": "0.0.33",
+    "@types/uuid": "^9.0.8",
+    "express": "^4.19.2",
+    "http-terminator": "^3.2.0",
+    "jshint": "^2.13.6",
+    "nock": "~13",
+    "prettier": "^3.2.5",
+    "semver": "^7.6.0",
+    "tap": "^18.7.2",
+    "ts-node": "^10.9.2",
     "tslint": "^6.1.1",
     "tslint-config-prettier": "^1.18.0",
-    "tslint-react": "^4.2.0",
```

Comment:
Note that `tslint` is deprecated and should be replaced with `eslint`, I plan to do that in the future.